src/v/kafka/server/snc_quota_manager.cc

```diff
@@ -59,6 +61,72 @@ to_soa_layout(const std::vector<ingress_egress_state<T>>& v) {
 
 namespace {
 
+const auto direction_label = ss::metrics::label("direction");
+const auto label_ingress = direction_label("ingress");
+const auto label_egress = direction_label("egress");
```

Comment:
maybe either move these into the function using them (or define as a constexpr string_view) so we don't have the clang tidy warnings.

Reply:
`ss::metrics::label` has no constexpr ctors, so making them static const instead

Reply:
that'll still produce a clang tidy warning. better to use string view or move them into the function.

Reply:
yep sorry I meant to say I will make them `static const` inside the function. Is that ok for clang-tidy?

Reply:
yep then it has a place to raise an exception in allocation were to fail.